src/components/ui/OrganisationCard/index.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { DARK_GRAY } from 'settings/__color';
+import { fontSizes } from 'settings/__fonts';
+import __layouts from 'settings/__layouts';
+import styled from '@emotion/styled';
+
+export interface OrganisationCardProps {
+  img: string;
+  name: string;
+  role: string;
+  date: string;
+}
+
+export default function OrganisationCard(props: OrganisationCardProps): React.ReactElement<OrganisationCardProps> {
+  const { img, name, role, date } = props;
+  return (
+    <OrganisationCard.Wrapper>
+      <OrganisationCard.Content>
+        <div className="organisation-image">
+          <img src={img} alt="organisation image" />
+        </div>
+        <div className="organisation-name">
+          <h2>{name}</h2>
+          <div className="organisation-role">
+            <div>
+              <div className="role">
+                <p>
+                  <i className="icon ion-ios-person" /> &nbsp; {role}
+                </p>
+              </div>
+              <div className="date">
+                <p className="move-up">
+                  <i className="icon ion-md-calendar" /> &nbsp; {date}
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </OrganisationCard.Content>
+    </OrganisationCard.Wrapper>
+  );
+}
+
+OrganisationCard.Wrapper = styled.div`
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.1);
+  border-radius: 3px;
+`;
+
+OrganisationCard.Content = styled.section`
+  width: 95vw;
+  display: flex;
+  height: 33vh;
+
+  .organisation-image {
+    height: 100%;
+    width: 28vw;
+    margin-right: 20px;
+    overflow: hidden;
+
+    img {
+      height: 100%;
+    }
+  }
+  .organisation-name {
+    padding: 0 10px;
+    margin-right: 20px;
+  }
+
+  .organisation-role {
+    line-height: 12px;
+  }
+
+  p {
+    color: ${DARK_GRAY};
+  }
+  @media (max-width: ${__layouts.xLg}) {
+    height: 16vh;
+  }
+
+  @media (max-width: ${__layouts.md}) {
+    height: 25vh;
```

Comment:
This seems to cause some problems. Once you remove this the page is parfect on all phones
<img width="1440" alt="Screenshot 2020-01-13 at 9 55 37 PM" src="https://user-images.githubusercontent.com/38565349/72291417-9a769380-364f-11ea-88d7-b9850b80f241.png">
<img width="1440" alt="Screenshot 2020-01-13 at 9 55 25 PM" src="https://user-images.githubusercontent.com/38565349/72291418-9a769380-364f-11ea-8eb8-36f5bb23d257.png">

Reply:
Oh I see. Thank you. Implementing the feedback asap